src/main/java/com/google/devtools/build/lib/unix/ProcMeminfoParser.java

```diff
@@ -23,6 +23,7 @@
 import java.nio.charset.Charset;
 import java.util.List;
 import java.util.Map;
+import java.util.HashMap;
 
 /**
  * Parse and return information from /proc/meminfo.
```

Comment:
Can you add documentation here what happens if there are duplicate entries?

Reply:
updated